src/gordon_gcp/plugins/janitor/reconciler.py

```diff
@@ -282,11 +282,10 @@ def _remove_soa_and_root_ns(self, zone, rrsets):
         Returns:
             tuple[list[rrset], list[rrset]]: The missing and extra rrset lists.
         """
-        desired_rrsets = [
-            gdns.GCPResourceRecordSet(**record) for record in rrsets
-        ]
+        desired_rrsets = gdns.GDNSClient.get_rrsets_as_objects(rrsets)
```

Comment:
just wondering, why did you chose to add `get_rrsets_as_objects` as a static method then call it as such here? this class already has the `dns_client` available, so could just be `self.dns_client.get_rrsets_as_objects(rrsets)`? apologies if i'm missing something

Reply:
So my usual MO was to make methods that didn't depend on instance variables static.  It indicates that separation.  FWIW, tho, the new style guide hates static methods and tells you to make either instance-level methods or module-level functions.

Reply:
Yeah, it was basically what ocsw said, but I don't mind doing it as an instance method if you folks prefer. It makes it a little more flexible if we could ever imagine some config info being related to the process, in the same way I left `get_managed_zone` as an instance method, because I thought it was plausible want to drop the auto-translation and instead pass in a config when initializing the dns client that contains a map of dns->managed zones at some point.

Reply:
I'm ok either way.